src/scenedit/scen.menus.mac.mm

```diff
@@ -20,7 +20,7 @@
 extern cUndoList undo_list;
 
 @interface MenuHandler : NSObject
--(void) menuChoice:(id) sender;
+-(IBAction) menuChoice:(id) sender;
```

Comment:
Wait, should these really be IBAction? It's not like I'm directly linking them to a menuitem in the interface builder... (I mean, IBAction is defined to be void, so it doesn't matter that much, but...)

Reply:
It does not need to be an `IBAction` no.